### DISCUSSION TITLE
Add number entry box to sliders

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#16932] [Ideas]
![osu!_aw2p1isZ56](https://user-images.githubusercontent.com/22729896/154856596-ed509cdf-bb62-4fd4-8278-f8239b92ea28.png)

Since I am playing osu with a graphics tablet these sliders are kind of hard to get to the correct values and overing on the slider and pressing arrow keys sometime leads to changing song or not moving the slider.

I would prefer if there was just a box where I could enter the value myself.

It would also make the values always visible without a tooltip which is better in my oppinion.

**Comment by peppy** [comment]
Agree with both points. Should be applied to all slider controls though.

**Comment by KoleckOLP** [comment]
I hope this is planned or something, should I make an issue about this?

**Reply by bdach** [reply]
no. we are aware, pending redesigns have this
